docker-entrypoint.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
This is basically from k8s-vm. I did some structural modifications and the way we invoke it will slightly differ.